.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
   - sudo mkdir -p /etc/montagu/api/ && sudo touch /etc/montagu/api/go_signal
   - ./src/gradlew -p src :startDatabase :startOrderlyWeb :startTaskQueue :startTestAPI
 script:
-  - ./src/gradlew -p src :testLibrary :stopTaskQueue :stopTestAPI
```

Comment:
we now need the task q to be running during blackbox tests, so don't stop it here